src/main/java/us/jonathans/interface_adapter/get_leaderboard/GetLeaderboardController.java

```diff
@@ -10,7 +12,7 @@ public GetLeaderboardController(GetLeaderboardInputBoundary getLeaderboardIntera
         this.getLeaderboardInteractor = getLeaderboardInteractor;
     }
 
-    public void execute(){
+    public void execute() throws IOException, InterruptedException {
```

Comment:
Do not propagate exceptions to the controller layer since they originate from the data access layer. Handle them at a lower level (with try catch).